src/steps/nodes/converters.ts

```diff
@@ -4,6 +4,32 @@ import {
   parseTimePropertyValue,
 } from '@jupiterone/integration-sdk-core';
 import { Entities } from '../constants';
+import { V1NodeAddress } from '@kubernetes/client-node';
+
+function getAddressesByType(type: string, addresses?: V1NodeAddress[]) {
+  if (!addresses) {
+    return undefined;
+  }
+
+  return addresses
+    .filter((address) => address.type.toLowerCase() === type.toLowerCase())
+    .map((address) => address.address);
+}
+
+function getAddressByType(type: string, addresses?: V1NodeAddress[]) {
+  if (!addresses) {
+    return undefined;
+  }
+
+  const address = addresses.find(
+    (address) => address.type.toLowerCase() === type.toLowerCase(),
+  );
+  if (!address) {
+    return undefined;
+  }
+
+  return address.address;
```

Comment:
Super small nit, but I would have done the following to completely remove the above `if` condition: 

```suggestion
  return address?.address;
```

Reply:
Good nit :+1: